tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -6255,6 +6257,7 @@ declare namespace ts {
         Insert = "insert",
         Remove = "remove"
     }
+    /** @deprecated - consider using EditorSettings instead */
```

Comment:
Could be a bad diff, but I'm seeing `EditorOptions` deprecated?  What changed?

Reply:
It was marked deprecated before, but with a normal comment (`/* */`) as opposed to JSDoc (`/** */`), just a drive-by fix.